src/modules/presets.js

```diff
@@ -0,0 +1,33 @@
+function applyPreset(preset) {
+  const result = {};
+  switch (preset) {
+    case 'dial-once': {
+      if (['staging', 'production'].includes(process.env.NODE_ENV)) {
+        process.env.CONSOLE_LOGGING = 'false';
```

Comment:
I can change the logic, but this was done because envs have a higher priority and if you use a preset ``dial-once``, that would set ``CONSOLE_LOGGING = false`` in ``settings`` object, but you also accidentally have it as an env var and it is set to ``true``, you will loose a good part of time to understand that your one is what's overwriting the setting